data/packages.xml

```diff
@@ -486,8 +485,11 @@
       <pkgname>splix</pkgname>
     </feature>
     <feature name="firefox">
-        <pkgname conflicts="chromium,chromium-pepper-flash">firefox</pkgname>
+        <pkgname conflicts="chromium">firefox</pkgname>
+    </feature>
+    <feature name="flash">
         <pkgname conflicts="chromium,chromium-pepper-flash">flashplugin</pkgname>
+        <pkgname conflicts="firefox,flashplugin">chromium-pepper-flash</pkgname>
     </feature>
```

Comment:
@Wyn10 :
That's better but still does not work. I didn't explain it well before. See, if you tell Cnchi to install "flash", it will try to install flashplugin AND chromium-pepper-flash. That's how packages.xml works. Problem is the conflicts attrib. Cnchi won't install chromium and chromium-pepperflash because it conflicts with flashplugin, and then when it will try to install chromium-pepper-flash won't be able to do it either as it conflicts with firefox and flashplugin.

I think we can safely install flashplugin and chromium-pepper-flash if the user selects "flash", can't we? So simply removing the conflicts attribute in "flash" will do.

Don't worry I'll do it myself. Thanks!